.github/workflows/build-and-deploy-dev.yml

```diff
@@ -7,8 +7,23 @@ on:
     branches: [master]
 
 jobs:
+  upload-package-lock-json:
+    name: Upload package-lock.json from this repo
+    runs-on: ubuntu-latest
```

Comment:
This uploading job is somewhat cumbersome.  Hopefully, we can improve this in the future.